conf/modules_illumina.config

```diff
@@ -833,16 +833,15 @@ if (!params.skip_assembly) {
         if (!params.skip_blast) {
             process {
                 withName: '.*:.*:ASSEMBLY_SPADES:.*:BLAST_BLASTN' {
-                    ext.args = "-outfmt '6 stitle std slen qlen qcovs'"
+                    ext.args = "-outfmt '6 stitle staxids std slen qlen qcovs'"
                     publishDir = [
                         path: { "${params.outdir}/assembly/spades/${params.spades_mode}/blastn" },
-                        mode: params.publish_dir_mode,
-                        saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
+                        enabled: false
```

Comment:
It might not always be interesting to not publish this? What about an additional variable that controls this `--save_blast_unfiltered` ? 

If you agree make sure to keep `${meta.id}.filter.blastn`

Reply:
Unfiltered blast is always saved, but it does it in the other process, here is only unfiltered blast with no header.

Reply:
It is generated inside `modules/local/filter_blastn.nf`

Reply:
Aaah sorry! I missed those 2 new lines :D